services/src/test/java/edu/unc/lib/dl/cdr/services/rest/modify/AddContainerIT.java

```diff
@@ -128,6 +136,10 @@ public void testAddFolder() throws Exception {
         Map<String, Object> respMap = getMapFromResponse(result);
         assertEquals(collObj.getPid().getId(), respMap.get("pid"));
         assertEquals("create", respMap.get("action"));
+
+        PatronAccessDetails accessDetails = new PatronAccessDetails();
```

Comment:
Would be good to update the test method name to something like `testAddFolderStaffOnly`